src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -1776,7 +1776,8 @@ export class ZonePacketHandlers {
       if (
         !(
           entity instanceof ConstructionParentEntity ||
-          entity instanceof ConstructionChildEntity
+          entity instanceof ConstructionChildEntity ||
+          entity instanceof LootableConstructionEntity
```

Comment:
this won't affect other stuff ?

Reply:
Not from what I've tested, looks good so far 👍

Reply:
Did you test decay? Because objects not placed on a foundation will decay a lot faster